docs/topics/scale.md

```diff
@@ -28,16 +30,208 @@ This command will re-use the `apimodel.json` file inside the output directory as
+k8s-agentpool1-10367588-vmss000007    Ready       v1.18.3    Ubuntu 18.04.5 LTS    5.4.0-1025-azure
+k8s-agentpool1-10367588-vmss000008    Ready       v1.18.3    Ubuntu 18.04.5 LTS    5.4.0-1025-azure
+k8s-agentpool1-10367588-vmss000009    Ready       v1.18.3    Ubuntu 18.04.5 LTS    5.4.0-1025-azure
+k8s-agentpool1-10367588-vmss00000a    NotReady    v1.18.8    Ubuntu 18.04.5 LTS    5.4.0-1025-azure
+k8s-agentpool1-10367588-vmss00000b    NotReady    v1.18.8    Ubuntu 18.04.5 LTS    5.4.0-1025-azure
+```
+
+Shortly after the new nodes are `Ready`, and running `v1.18.8`:
+
+```
+$ k get nodes -o wide
+NAME                                 STATUS   ROLES    AGE   VERSION   INTERNAL-IP    EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION     CONTAINER-RUNTIME
+k8s-agentpool1-10367588-vmss000002   Ready    agent    49m   v1.18.3   10.240.0.96    <none>        Ubuntu 18.04.5 LTS   5.4.0-1025-azure   docker://19.3.12
+k8s-agentpool1-10367588-vmss000003   Ready    agent    49m   v1.18.3   10.240.0.127   <none>        Ubuntu 18.04.5 LTS   5.4.0-1025-azure   docker://19.3.12
+k8s-agentpool1-10367588-vmss000004   Ready    agent    50m   v1.18.3   10.240.0.158   <none>        Ubuntu 18.04.5 LTS   5.4.0-1025-azure   docker://19.3.12
+k8s-agentpool1-10367588-vmss000005   Ready    agent    50m   v1.18.3   10.240.0.189   <none>        Ubuntu 18.04.5 LTS   5.4.0-1025-azure   docker://19.3.12
+k8s-agentpool1-10367588-vmss000006   Ready    agent    50m   v1.18.3   10.240.0.220   <none>        Ubuntu 18.04.5 LTS   5.4.0-1025-azure   docker://19.3.12
+k8s-agentpool1-10367588-vmss000007   Ready    agent    50m   v1.18.3   10.240.0.251   <none>        Ubuntu 18.04.5 LTS   5.4.0-1025-azure   docker://19.3.12
+k8s-agentpool1-10367588-vmss000008   Ready    agent    50m   v1.18.3   10.240.1.26    <none>        Ubuntu 18.04.5 LTS   5.4.0-1025-azure   docker://19.3.12
+k8s-agentpool1-10367588-vmss000009   Ready    agent    50m   v1.18.3   10.240.1.57    <none>        Ubuntu 18.04.5 LTS   5.4.0-1025-azure   docker://19.3.12
+k8s-agentpool1-10367588-vmss00000a   Ready    agent    65s   v1.18.8   10.240.0.34    <none>        Ubuntu 18.04.5 LTS   5.4.0-1025-azure   docker://19.3.12
+k8s-agentpool1-10367588-vmss00000b   Ready    agent    68s   v1.18.8   10.240.0.65    <none>        Ubuntu 18.04.5 LTS   5.4.0-1025-azure   docker://19.3.12
+k8s-master-10367588-0                Ready    master   55m   v1.18.3   10.255.255.5   <none>        Ubuntu 18.04.4 LTS   5.3.0-1022-azure   docker://3.0.12+azure
+```
+
+In summary, by updating your API model to require the latest Kubernetes version, we produce an ARM template deployment that is able to be executed successully without traversing outside the VNET. As before, we've lost strict homogeneity, but because Kubernetes guarantees functional compatibility within a minor release channel (no breaking changes with patch releases), we now have an arguably more operationally stable cluster running the latest validated bits.
```

Comment:
```suggestion
In summary, by updating your API model to require the latest Kubernetes version, we produce an ARM template deployment that is able to be executed successfully without traversing outside the VNET. As before, we've lost strict homogeneity, but because Kubernetes guarantees functional compatibility within a minor release channel (no breaking changes with patch releases), we now have an operationally stable cluster running the latest validated bits.
```